administrator/components/com_admin/sql/updates/postgresql/4.1.0-2021-06-08.sql

```diff
@@ -22,5 +22,5 @@ CREATE INDEX "#__cookiemanager_cookies_idx_state" on "#__cookiemanager_cookies"
 CREATE INDEX "#__cookiemanager_cookies_idx_catid" on "#__cookiemanager_cookies" ("catid");
 CREATE INDEX "#__cookiemanager_cookies_idx_createdby" on "#__cookiemanager_cookies" ("created_by");
 
-INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder", "client_id", "enabled", "access", "protected", "locked", "manifest_cache", "params", "custom_data", "checked_out", "checked_out_time", "ordering", "state") VALUES
-(0, 'com_contact', 'component', 'com_contact', '', 1, 1, 1, 0, 1, '', '{"policylink":"","position":"bottom"}', '', 0, NULL, 0, 0);
+INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder", "client_id", "enabled", "access", "protected", "locked", "manifest_cache", "params", "custom_data", "ordering", "state") VALUES
+(0, 'com_cookiemanager', 'component', 'com_cookiemanager', '', 1, 1, 1, 0, 1, '', '{"policylink":"","position":"bottom"}', '', 0, 0);
```

Comment:
In the update SQL for mysql you don't have the `ordering` and `state` columns included but here you have. Is that by purpose? I think it should be consistent between both update SQL scripts.

Reply:
just removed ordering and state now it's same as for extensions and corrected it also

Reply:
Hmm, I see in base.sql it is different, too, between both database types . Not nice but not related to your works. Here i think both update SQL scripts should be consistent with each other, either the one or the other way.

Reply:
In base.sql for PostgreSQL have to add `ordering` and `state` also but not in base.sql for MySQL,
So I just add the same PostgreSQL base to PostgreSQL updates and MySQL base to MySQL updates.

Reply:
Yes I noticed. It's ok. But the installation with postgresql seems to fail in Drone.

Reply:
Found the reason, see suggestions below.